_data/categories/meshes.yml

```diff
@@ -10,6 +10,19 @@
   prometheus: "No, Replaced by Harmony Controller"
   tracing: "Yes"
   encryption: "Yes"
+  
+  - name: App Mesh
+  desc: A multi-service mesh management plane to enable adoption, operation and development of service meshes and services running on them
+  link: https://github.com/layer5io/meshery-app-mesh
```

Comment:
We'll want for the link to point to AWS App Mesh, not the Meshery adapter for App Mesh.